scraper/docsearch-config.json

```diff
@@ -1,11 +1,6 @@
 {
-  "index_name": "camino-docs",
-  "start_urls": [
-    {
-      "url": "https://docs.camino.foundation",
-      "tags": ["prod"]
-    }
-  ],
+  "index_name": "camino-docs-default",
+  "start_urls": ["https://docs.camino.network"],
```

Comment:
what about dev.docs.camino.network ?

Reply:
This is replaced with a `sed` command to `host.docker.internal` in the workflow's last step and then scraped from that local copy. So, if it runs for `main`, it's `main`'s web site that is scraped and if `dev` branch it's `dev`'s site.